src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import "./Carousel.css";
+import { BsArrowLeftCircleFill, BsArrowRightCircleFill } from "react-icons/bs";
+
+interface ImageProps {
+  imgList: { url: string; alt: string }[];
```

Comment:
it's ok but try to introduce a separate interface for:
`{ url: string; alt: string }`
once you have a need for an object in your type declaration, might mean that you need an interface. That would allow you break down your code nicely and reuse the new interface in other places if needed